### DISCUSSION TITLE
Сustom scripts available ip in prefix.

### DISCUSSION THREAD

**Discussion by zhabchenko** [netbox-community/netbox#13288] [Q&A]
Hi all! As for the ObjectVar field, get the available addresses in the prefix.

**Comment by kkthxbye-code** [comment]
That's not really possible out of the box as far as I know. Why does the user need to choose the IP? Is automatically assigning a free one not sufficient?

**Reply by zhabchenko** [reply]
Unfortunately not, he needs to choose which ip to start with(
